File: test-serenity2.Web/Modules/ServerTypes/Games/LogsForm.ts
import { StringEditor, IntegerEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface LogsForm {
    EntityName: StringEditor;
    EntityId: IntegerEditor;
    Action: StringEditor;
}

export class LogsForm extends PrefixedContext {
    static readonly formKey = 'Games.Logs';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!LogsForm.init)  {
            LogsForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;

            initFormType(LogsForm, [
                'EntityName', w0,
                'EntityId', w1,
                'Action', w0
            ]);
        }
    }
}

File: test-serenity2.Web/Modules/Games/Logs/LogsForm.cs
using Serenity.ComponentModel;

namespace test_serenity2.Games.Forms;

[FormScript("Games.Logs")]
[BasedOnRow(typeof(LogsRow), CheckNames = true)]
public class LogsForm
{
    public string EntityName { get; set; }
    public int EntityId { get; set; }
    public string Action { get; set; }
}